packages/theme-core-plugin--sass/sass.js

```diff
@@ -76,31 +117,8 @@ module.exports = (userConfig) => {
   }
 
   clean.description = 'Clean compiled CSS';
-
   tasks.clean = clean;
 
-  function validateCss(errorShouldExit) {
```

Comment:
Side note on this one: I had to move this function further up so we're not throwing any eslint errors (calling a function before it's defined) based on [this change here](https://github.com/drupal-pattern-lab/theme-tools/pull/3/files#diff-bdf5ce03febacb87e995b8e19771ad9aR103) -- automatically running Stylelint initially when the code gets compiled IF linting is enabled.

There's a couple changes to this linting function as well (ex. adding Gulp Plumber) but just wanted to give a heads up on this